DIRECTORY.md

```diff
@@ -51,6 +51,7 @@
   * [Index Of Rightmost Set Bit](bit_manipulation/index_of_rightmost_set_bit.py)
   * [Is Even](bit_manipulation/is_even.py)
   * [Is Power Of Two](bit_manipulation/is_power_of_two.py)
+  * [Is Power Of Two](bit_manipulation/largest_pow_of_two_less_than_or_equal_to_a_number.py)
```

Comment:
Please __DO NOT__ edit `DIRECTORY.md`. We have a workflow to do this for us.

Reply:
Understood, I'll revert this change.